components/search_bar/search_bar.jsx

```diff
@@ -143,22 +143,22 @@ export default class SearchBar extends React.Component {
     }
 
     renderHintPopover(helpClass) {
-        if (!this.props.isCommentsPage && Utils.isMobile() && this.state.isPristine) {
-            return false;
+        if (!this.props.isCommentsPage && this.state.isPristine) {
+            return (
+                <Popover
+                    id='searchbar-help-popup'
+                    placement='bottom'
+                    className={helpClass}
+                >
+                    <FormattedHTMLMessage
+                        id='search_bar.usage'
+                        defaultMessage='<h4>Search Options</h4><ul><li><span>Use </span><b>"quotation marks"</b><span> to search for phrases</span></li><li><span>Use </span><b>from:</b><span> to find posts from specific users and </span><b>in:</b><span> to find posts in specific channels</span></li></ul>'
+                    />
```

Comment:
Not sure if we should make an exception in using `FormattedHTMLMessage` or we need to find a way so that `FormattedMarkdownMessage` will work here.

Reply:
@saturninoabril No new changes here, I've just changed the if statement, that markup was already there. So not sure if that needs to be changed in this PR.

Reply:
Yup, I know and I don't think we need to change in this PR. I just brought it up since we usually upgrade the code (by getting rid off FormattedHTMLMessage in favor of FormattedMarkdownMessage whenever possible) if we touches part of the code and we have the opportunity to do so.

Reply:
Sounds cool.